be/src/olap/storage_engine.cpp

```diff
@@ -1035,4 +1035,76 @@ void StorageEngine::create_base_compaction(TabletSharedPtr best_tablet,
     base_compaction.reset(new BaseCompaction(best_tablet, tracker_label, _compaction_mem_tracker));
 }
 
+// Return json:
+// {
+//   "CumulativeCompaction": {
+//          "/home/disk1" : [10001, 10002],
```

Comment:
Can we add segment number that be merged in this compaction task for each thread? Of course, we could add the segment number info in another PR.